ovh/resource_vps_gen.go

```diff
@@ -68,6 +69,12 @@ func VpsResourceSchema(ctx context.Context) schema.Schema {
 			Description:         "IAM resource metadata",
 			MarkdownDescription: "IAM resource metadata",
 		},
+		"image_id": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Optional:            true,
+			Description:         "Id of the VPS. Image fetched in /images list",
```

Comment:
```suggestion
			Description:         "Id of the image to install on the VPS
```

same for `MarkdownDescription`